### DISCUSSION TITLE
Indexing status

### DISCUSSION THREAD

**Discussion by yasiryagi** [subquery/subql#976] [Frontier TestNet - Bug Reporting]
The indexing status displayed as below even after refreshing the page.

![13](https://user-images.githubusercontent.com/4862448/166913070-a578d29f-c68b-44d4-b1e7-1582977440fd.PNG)

But after browsing into the project and back, it display the below

![14](https://user-images.githubusercontent.com/4862448/166913086-464c9f7e-1853-49dd-8f93-2a867e8680e1.PNG)

**Comment by jim-counter** [comment]
I see this too - seems to be almost random when I keep refreshing.

**Reply by Staking7pc** [reply]
It happens to us too. Just the first time when the admin page is loaded . Once you enter some project and go back to main page the status updates to the expected value "Ready"

**Comment by mzxyz** [comment]
Thanks for reporting this issue, added to our backlog.

**Comment by mzxyz** [comment]
Resolved in `coordinator:v0.1.10`

**Reply by jim-counter** [reply]
Just tested - I'm now unable to reproduce on this build. Thanks!
